### PR TITLE
Add release notes for 6.1.0

### PR DESCRIPTION
**Pull request by jsvd**
This list was compiled from https://github.com/elastic/logstash/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Av6.1.0+is%3Aclosed+ and the plugin's changelogs bumped since 6.0.0/6.0.1

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -3,40 +3,38 @@
...
-  configurations directly in Kibana without having to restart Logstash.
-* Users can now visualize the Logstash configuration pipeline as part of the {xpack} monitoring feature.
-
-==== Plugins
-
-* `GeoIP Filter`: You can now use MaxMind's commercial database to get enriched Geo information. ASN data can be
-  obtained via the GeoIP2-ISP database.
+[[logstash-6-1-0]]
+=== Logstash 6.1.0 Release Notes
+* New experimental Java execution engine for Logstash pipelines. Off by default, it can be enabled with `--experimental-java-execution` ({lsissue}7950[Issue 7950]).
```
Comment (jakelandis) [inline docs/static/releasenotes.asciidoc]: Are the issues links supposed to render when you click the view button ?

Comment (jsvd) [reply docs/static/releasenotes.asciidoc]: not really, other examples here: https://github.com/elastic/logstash/blob/v5.5.2/docs/static/releasenotes.asciidoc

**Comment by jakelandis** [comment]
I browsed the core changes and didn't see any additional non-internal type changes users would care about.  From memory, the plugin list seems about right (it's a fairly short cycle for the plugins). 

LGTM

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -3,40 +3,38 @@
...
+*`Ruby`*:
+
+* Fixed concurrency issues with multiple worker threads that was caused by a (https://github.com/jruby/jruby/issues/4868[JRuby issue]).
+* Added file based Ruby script support as an alternative to the existing inline option (https://github.com/logstash-plugins/logstash-filter-ruby/pull/35[#35]).
+
+==== Output Plugins
+
+*`Elasticsearch`*:
+
+* Ignore event's `type` field for the purpose of setting the document's `_type` when indexing to Elasticsearch 6.x or above (https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/712[#712]).
```
Comment (dedemorton) [inline docs/static/releasenotes.asciidoc]: Would change to say:

When indexing to Elasticsearch 6.x or above, Logstash ignores the event's type field and no longer uses it to set the document's `_type`.

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -3,40 +3,38 @@
...
-
-==== Plugins
-
-* `GeoIP Filter`: You can now use MaxMind's commercial database to get enriched Geo information. ASN data can be
-  obtained via the GeoIP2-ISP database.
+[[logstash-6-1-0]]
+=== Logstash 6.1.0 Release Notes
+* New experimental Java execution engine for Logstash pipelines. Off by default, it can be enabled with `--experimental-java-execution` ({lsissue}7950[Issue 7950]).
+* Added support for changing the <<configuring-persistent-queues,page capacity>> for an existing queue ({lsissue}8628[Issue 8628]).
+* {lsissue}7692[Many], {lsissue}8776[many], {lsissue}8577[many], {lsissue}8446[many], {lsissue}8333[many], {lsissue}8163[many], {lsissue}8103[many], {lsissue}8087[many] (did I say {lsissue}7691[many]?) improvements in pipeline execution performance and memory efficiency.
```
[the remark this answers is not in the document]
Comment (jsvd) [reply docs/static/releasenotes.asciidoc]: hahah you're right, I failed at being funny with added cost of reduced discovery. Will take your suggestion.

**Comment by jsvd** [comment]
@dedemorton I've updated the content according to your remarks

**Pull request merged**